AppCenterCrashes/AppCenterCrashesTests/MSCrashesCategoryTests.m

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAppCenterInternal.h"
+#import "MSCrashesCategory.h"
+#import "MSCrashesPrivate.h"
+#import "MSMockUserDefaults.h"
+#import "MSTestFrameworks.h"
+#import "MSUtility+Application.h"
+
+#if TARGET_OS_OSX
+
+@interface NSApplication (CrashException)
+
+- (void)ms_reportException:(NSException *)exception;
+- (void)ms_sendEvent:(NSEvent *)theEvent;
+
+@end
+
+#endif
+
+static NSException *lastException;
```

Comment:
Why is it static? Because there is only one unit test?

Reply:
static here means that it will NOT create symbol for it to access from other files

Reply:
My point is that this will be used more than one method in this class once you add more unit tests. You don't see any of side effects by using static variable?

Reply:
If you are saying about `static` modifier - it's better to have it to avoid possible name conflicts at linking.

if about global var in general - we need it to have a pointer to pure c function and it's required to check the behavior. There is no way to mock it locally